packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -647,6 +647,15 @@ class FlutterPlugin implements Plugin<Project> {
                 extraGenSnapshotOptions extraGenSnapshotOptionsValue
                 splitDebugInfo splitDebugInfoValue
                 treeShakeIcons treeShakeIconsOptionsValue
+                doLast {
+                    project.exec {
+                        if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+                            commandLine('cmd', '/c', 'attrib -r ${assetsDirectory}')
```

Comment:
I just realized that this does not run recursively. We need to run this instead:
`commandLine('cmd', '/c', 'attrib -r ${assetsDirectory}/* /s')`